docs/tutorial/troubleshooting/migrate_pool_by_migrating_disks.md

```diff
@@ -0,0 +1,314 @@
+# Pool Migration when underlying disks were migrated to different node
+
+## Intro
+- There can be situations where your Kubernetes cluster scales down and scales up again(To save cost). This can cause nodes to come with a different hostName and node name than the previous existing nodes, meaning the nodes that have come up are new nodes and not the same as previous nodes that earlier existed.
+- When the new nodes come up, the disks that were attached to the older nodes now get attached to the newer nodes.
+
+Consider and example where you have following nodes in a Kubernetes cluster with disks attached it:
+ 
+    Worker-1 (bd-1w1, bd-2w1 are attached to worker1) 
```

Comment:
n_p: I recently learned that   bd-1(w1) is a good representation. 
Everywhere else we will be using this representation and using at some places. 
This will make the doc look homogenous.